src/assets/styles/base.scss

```diff
@@ -69,11 +69,18 @@ h2 {
 }
 
 h3 {
-  font-size: 0.95rem;   
+  font-size: 0.95rem;
```

Comment:
Really? 1rem was too big?

Reply:
lol i didn't change this! Want me to move this to `1rem`?

Reply:
Is there an actual reason it's almost-but-not-quite 1 rem?

Yeah, change it. It looks too weird. :)

Reply:
Or if there's a valid reason for this, it needs a comment!

Reply:
lol i've no idea, it was set at .95 initially. Updated to 1 😁